content/help/faq/security/k8s-health-checks.md

```diff
@@ -2,23 +2,16 @@
 title: How can I use Kubernetes liveness and readiness for pod health checks when mutual TLS is enabled?
 weight: 50
 ---
-If mutual TLS is enabled, http and tcp health checks from the kubelet will
-not work since the kubelet does not have Istio-issued certificates.
 
-As of the Istio 1.0 release, we support the [`PERMISSIVE` mode](/docs/tasks/security/mtls-migration)
-for Istio services so they can accept both http and mutual TLS traffic
-when this mode is turned on. This can solve the health checking issue.
-Please keep in mind that mutual TLS is not enforced since others can
-communicate with the service with http traffic.
+If mutual TLS is enabled, http and tcp health checks from the kubelet will not work since the kubelet does not have Istio-issued certificates.
```

Comment:
```suggestion
If mutual TLS is enabled, HTTP and TCP health checks from the kubelet will not work without modification, since the kubelet does not have Istio-issued certificates.
```